bin/inference/pycbc_inference

```diff
@@ -84,6 +84,8 @@ use_mpi, size, rank = pycbc.pool.use_mpi(opts.use_mpi, log=False)
 if use_mpi:
     opts.verbose &= rank == 0
 pycbc.init_logging(opts.verbose)
+if use_mpi and rank != 0:
```

Comment:
I think this makes lines 84-85 superfluous now. As the docs seem to indicate disabling in this way is cumulative, i.e. that level and less critical ones are all disabled.

Reply:
@ahnitz Sorry for forgetting about this. I've removed the previous `if` block.